aw-datastore/src/datastore.rs

```diff
@@ -581,4 +598,84 @@ impl DatastoreInstance {
 
         return Ok(count);
     }
+
+    pub fn insert_value(&self, conn: &Connection, key: &str, data: &str)
```

Comment:
I'd prefer if these function names said "key_value" rather than just "value".
For example, insert_value does not necessarily only set the value but also sets a new key if it does not yet exist so calling it just "value" doesn't really make any sense.
Same thing goes for "delete" and all the others.

Reply:
True. I I'll change that.